apps/docs/src/basics/motion.md

```diff
@@ -0,0 +1,35 @@
+# Motion
+
+Microanimations bring a user interface to life. However, careful handling is necessary here, because the minimalistic motto is
+
+::: tip Less is more!
+The user should not be distracted by elaborate animations. Instead, they should support the user in his workflow, provide feedback and make the entire application easier to understand.
+:::
+
+For this reason, onyx provides a selected set of animations that fulfills these requirements. Default animations are already part of the components.
+
+## Types of animations
```

Comment:
```suggestion
## Types of motions
```

Some of them are transitions instead if animations so maybe we should just call them motions here?